apps/avatax/src/modules/taxes/get-active-connection-service.ts

```diff
@@ -38,6 +41,21 @@ export function getActiveConnectionService(
     return err(appConfigResult.error);
   }
 
+  /**
+   * This class is called on every webhook and config is resolved here for now. Before we refactor, we log here
+   * resolved configuration.
+   * Since this is constant refactor, wrap it in try/catch so broken log doesn't break the app.
+   * In the future we should extract configuration retrieval higher and pass in down the app's logic.
+   */
+  try {
+    new AppConfigurationLogger(logger).logConfiguration(
+      AppConfig.createFromParsedConfig(appConfigResult.value),
+      channelSlug,
+    );
+  } catch (e) {
+    captureException(new Error("Failed to log configuration metric", { cause: e }));
```

Comment:
**Question:** Maybe we can have `LogConfigurationMeticError` here so we have nice grouping in Sentry?

Reply:
done